src/styles/media.scss

```diff
@@ -0,0 +1,26 @@
+.bitmap {
+  height: 33px;
+  width: 33px;
+}
+
+.small-icon {
+  height: 16px;
+  object-fit: contain;
+  width: 17px;
+}
+
+.small-icon-grey {
+  height: 18px;
+  object-fit: contain;
+  opacity: 0.5;
+  width: 18px;
+}
+
+.avatar {
+  background-color: #2fbcf7;
+  border-radius: 50%;
+  height: 124px;
+  object-fit: contain;
+  padding: 2rem;
```

Comment:
Keep consistent with the units, use `rem` or `px` but not both. I prefer to use `rem`  but I saw that there are more `px` units, so it's your call